samples/twilio/program.py

```diff
@@ -0,0 +1,58 @@
+"""This program demonstrates AutoKitteh's Twilio integration.
+
+This program implements a single entry-point function triggered by an
+HTTP GET request event, as defined in the "autokitteh.yaml" manifest file.
+
+API details:
+- Messaging API overview: https://www.twilio.com/docs/messaging/api
+- Voice API overview: https://www.twilio.com/docs/voice/api
+
+It also demonstrates using constant values which are set for each
+AutoKitteh environment in the "autokitteh.yaml" manifest file.
+"""
+
+import os
+
+from autokitteh.twilio import twilio_client
+
+FROM_PHONE_NUMBER = os.getenv("FROM_PHONE_NUMBER")
+
+t = twilio_client("twilio_conn")
+
+
+def on_http_get(event):
+    """Entry-point for workflow.
+
+    This function is triggered by an HTTP GET request event and is used to
+    send SMS and WhatsApp messages via Twilio.
+
+    Example usage:
+    curl "http://localhost:9980/webhooks/<webhook_slug>?to=15555555"
+
+    The phone number to send the message to must be provided in the query
+    parameter 'to'. The message will be sent both as an SMS and a WhatsApp
+    message to the specified number.
+
+    Args:
+        event (object): An event object containing the request data.
+    """
+    to = event.data["url"]["query"]["to"]
+    try:
```

Comment:
this try except is too wide. catch specific exception on smaller scopes.

Reply:
I removed the try except entirely. I could do a try except around each message sending, but I'm not sure it's that necessary here?